src/server/pfs/server/server_test.go

```diff
@@ -1652,6 +1652,64 @@ func TestPutFileNullCharacter(t *testing.T) {
 	require.YesError(t, err)
 }
 
+func TestArchiveCommit(t *testing.T) {
+	t.Parallel()
+	client, _ := getClientAndServer(t)
+
+	repo1 := "TestArchiveCommit"
+	repo2 := "TestBlockveCommit2"
```

Comment:
I do know you like to have strings of the same length :-P but if so, then I guess you don't need the '2'